saleor/core/templatetags/shop.py

```diff
@@ -23,5 +23,7 @@ def menu(context, site_menu=None, horizontal=False):
         return
     menus = context[NAVIGATION_CONTEXT_NAME]
     menu = next((menu for menu in menus if menu.pk == site_menu.pk), None)
-    menu_items = menu.items.all() if menu else None
+    if not menu:
+        return
+    menu_items = filter(lambda i: i.parent_id is None, menu.items.all())
```

Comment:
Personally I prefer list comprehension, it's more readable

Reply:
Sure, no worries